alexandria/encoding/cairo_project.toml

```diff
@@ -0,0 +1,3 @@
+[crate_roots]
+alexandria_encoding = "src"
+alexandria_math = "../math/src"
```

Comment:
Does it need to be here ?

Reply:
It is useful as it uses some code from math such as pow and shl and shr